### PR TITLE
adding intro

### PR DESCRIPTION
**Pull request by TomBasha**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1 +1,3 @@
 new file
+
+This is just for testing purposes.
```
Comment (johnelbasha) [inline readme.md]: I think you need to give a bit more context. Can you explain what you are testing?

**Review by johnelbasha — CHANGES_REQUESTED** [review]
just one item to change.

File: readme.md (hunk 1 above)
Comment (TomBasha) [reply readme.md]: Got it

**Review by johnelbasha — APPROVED** [review]
great job!

**Pull request merged**